backend/graph/tools/vectordb_search.py

```
@@ -0,0 +1,85 @@
+    configurable = config.get("configurable", {})
+    vectorstore = configurable["vectorstore"]
+    sources = configurable["sources"]
+    reranker = configurable["reranker"]
+    pool = configurable.get("questdb_pool")
+
+    try:
+        with TimedMetric(pool, Metrics.DOC_RESPONSE_TIME.value):
+            chunks, available_sources, lang_code = retrieve_and_rerank(
+                query, vectorstore, reranker, sources
+            )
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)